CHANGELOG.md

```diff
@@ -1,3 +1,50 @@
+## v1.1.0 (2024-03-05)
+
+[Detailed Release Notes](https://github.com/cytomining/pycytominer/releases/tag/v1.1.0)
```

Comment:
```suggestion
[Detailed release notes](https://github.com/cytomining/pycytominer/releases/tag/v1.1.0)
```

Reply:
Could you fix this capitalization in `dev_tools/commitizen/CHANGELOG.md.j2` as well?

Reply:
Great points, both! I'll see that these get updated.